File: Bal/RtMidiRouterLib/MidiClient/FilterMidiDestinationPrivate.cpp
#include "FilterMidiDestinationPrivate.h"

/*[[[cog
import cog
from FilterMidiDestinationPrivate import classFilterMidiDestinationPrivate


cog.outl(classFilterMidiDestinationPrivate.getClassCpp(),
        dedent=True, trimblanklines=True)


]]] */
 FilterMidiDestinationPrivate:: FilterMidiDestinationPrivate(QObject *parent)
    : MidiRoutersFilter(parent)
{
    ctorClass();
}

//[[[end]]]

void FilterMidiDestinationPrivate::ctorClass() {
    m_filterType = FilterType::TO_MIDI_DESTINATION;
}

File: Bal/RtMidiRouterLib/MidiClient/FilterMidiDestinationPrivate.py
from property import create_prpt, PrptClass



ary = [
    create_prpt("QString", 'baseMidiRouteInput'),
    create_prpt("FilterType", 'filterType'),
]

enumClasss = [

]

classFilterMidiDestinationPrivate = PrptClass("FilterMidiDestinationPrivate", ary, enumClasss)
classFilterMidiDestinationPrivate.inhirit_from = "MidiRoutersFilter"
